pandas/io/excel/_base.py

```diff
@@ -520,7 +533,10 @@ def read_excel(
 
 class BaseExcelReader(metaclass=abc.ABCMeta):
     def __init__(
-        self, filepath_or_buffer, storage_options: StorageOptions = None
+        self,
+        filepath_or_buffer,
+        storage_options: StorageOptions = None,
+        **engine_kwargs,
```

Comment:
`engine_kwargs` should be treated consistently throughout - can you make this `engine_kwargs` instead of `**engine_kwargs` throughout

Reply:
Done!